content/doc/developer/tutorial-adopt/_create-a-pull-request.adoc

```diff
@@ -0,0 +1,35 @@
+////
+// This file is only meant to be included as a snippet in other
+// documents.  It provides the same text to multiple files so that we can
+// make a single change and have it apply to multiple files in the adopt
+// a plugin tutorial.
+////
+
+== Create a pull request
+
+Commit that change:
+
+[subs="attributes"]
+``` bash
+$ git clean -f
+$ git add .
+$ git commit -m "{task-description}"
+```
+
+Push the change to GitHub:
+
+[subs="attributes"]
+``` bash
+$ git push origin --set-upstream {task-identifier}
```

Comment:
Beware that naming of remotes is not really standardized. https://github.com/cli/cli/issues/582#issuecomment-628077873